azure-pipelines/templates/test-job.yml

```diff
@@ -34,6 +34,8 @@ jobs:
     inputs:
       targetType: inline
       script: |
+        $pwd
+        Get-ChildItem
```

Comment:
Shouldn't it be removed from final PR?

Reply:
Already :)
Found right after pinging for review